README.md

```diff
@@ -190,12 +190,13 @@ Questions can be asked by raising an `Issue`.
 - [Joykishan Sharma](https://github.com/JoykishanSharma)
 - [Kabir Jain](https://github.com/kkkkkabir)
 - [Kehinde Adeleke](https://github.com/adeleke5140)
+- [Khalid Khan Aka Pablochocobae](https://github.com/pablochocobae)
 - [Krishna R ](https://github.com/Krishna-Ravi)
 - [Lishu Gupta](https://github.com/lishugupta652)
 - [Lundbyit](https://github.com/Lundbyit)
 - [Mahnoor Anwar](https://github.com/Mahnoor-Anwar)
 - [Manasi Mulay](https://github.com/manasi-mulay)
-- [Manu Gupta](https://github.com/ManuGupta9780)
+- [Manu Gupta](https://github.com/ManuGupta9780)  
```

Comment:
As said [here](https://github.com/EddieHubCommunity/hacktoberfest-practice/pull/290#discussion_r584084228), remove the space.